gcp/observability/logging.go

```diff
@@ -364,6 +366,17 @@ func (bl *binaryLogger) GetMethodLogger(methodName string) iblog.MethodLogger {
 	return nil
 }
 
+// parseMethod splits service and method from the input. It expects format
+// "service/method".
+func parseMethod(method string) (string, string, error) {
+	pos := strings.LastIndex(method, "/")
```

Comment:
Nit: maybe `Index` is a little less unusual than `LastIndex` since we know there is exactly one slash.

Reply:
Ah good point already validated by this point. Switched.